raiden/lightclient/handlers/light_client_service.py

```diff
@@ -64,6 +64,12 @@ def apply_message_order_filter(cls, message: LightClientProtocolMessage, msg_ord
     def get_light_client_payment(cls, payment_id, storage: SerializedSQLiteStorage):
         payment = storage.get_light_client_payment(payment_id)
         if payment:
-            payment = LightClientPayment(payment[3], payment[4], payment[5], int(payment[6]), payment[7],
```

Comment:
No the payment table didn't change, the only thing i did here was to put the named parameters, the order then is not important since you use the name of the parameter, actually the identifier is not the first param if you don't use named parameters, is the boolean is_lc_initiator.

Reply:
cool, thanks.